### DISCUSSION TITLE
How to keep hands tied overlay+ragdoll

### DISCUSSION THREAD

**Discussion by deafstoryteller** [Sixze/ALS-Refactored#528] [Q&A]
When my character is knocked back/ragdolled it stops having its hands tied- how would I keep the hands tied while during and after ragdolling?

I assume I'd also need to make a custom animation of getting up with my hands still tied?

**Comment by GStreetGames** [comment]
Try not rag-dolling the arms by setting all bodies below to not simulate physics.

**Comment by Magic-Summer** [comment]
Try just setting simulate physics (ragdoll) on the legs. This may easily give you the desired visual effect.
I assume getting up without arms would force tuning into the back first, so you would need different getting-up logic, and a different sit-up-and-stand animation montage, instead of the usual getting-up montage. Good luck!
